docs/migration_2_to_3.rst

```diff
@@ -149,3 +149,16 @@ Telegram API Server
 
 - The `server` parameter has been moved from the `Bot` instance to `api` in `BaseSession`.
 - The constant `aiogram.bot.api.TELEGRAM_PRODUCTION` has been moved to `aiogram.client.telegram.PRODUCTION`.
+
+
+Telegram objects transformation (to dict, to json, from json)
+===========================================
+
+- Methods :class:`TelegramObject.to_object()`, :class:`TelegramObject.to_json()` and :class:`TelegramObject.to_python()` 
+  have been removed due to the use of `pydantic <https://docs.pydantic.dev/>`_ models.
+- :class:`TelegramObject.model_validate_json()` should be used instead of :class:`TelegramObject.to_object()` 
```

Comment:
`to_object` should be replaced by `model_dump()`, not by `model_validate_json`, isn't it?

Reply:
model_dump() creates a dictionary representation of an object, and to_object() creates an instance of a class from a dictionary representation, if i am not mistaken. So model_dump() is a reverse for to_object(). 
I think model_validate() should be used here, instead of model_validate_json() in order to create an object from a dict, not a json string. 
Here is some code example, that ive just checked, 
```
json_string = '{serialized Message object}'
        
dictionary = json.loads(json_string)
        
print(type(Message.model_validate(dictionary)))
# output: <class 'aiogram.types.message.Message'>
```